src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction() {
+
+    }

```

Comment:
There's an empty constructor that is not being used. If there's no specific need for it, consider removing it to keep the code clean.